src/main/java/com/faforever/client/player/PlayerService.java

```diff
@@ -194,6 +210,26 @@ void createOrUpdatePlayerForPlayerInfo(@NotNull com.faforever.commons.lobby.Play
     resetIdleTime(player);
   }
 
+  public boolean containsNote(PlayerBean player) {
+    return playerNotes.containsKey(player.getId());
+  }
```

Comment:
Couldn't we just check if player.getNote() is blank since the player note should be set on all playerBean initialization?